### PR TITLE
Developer info setup

### PR DESCRIPTION
**Pull request by sametsJR**
Possibility to add skills for developer. Generate simple users controller

### CODE REVIEW

File: test/controllers/users_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```
Comment (RGBD) [inline test/controllers/users_controller_test.rb]: Remove test for now

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  get 'users/index'
```
Comment (RGBD) [inline config/routes.rb]: resources :users, only: :index

File: app/views/users/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<% if user_signed_in? %>
+
+<h1>All users</h1>
+
+  <% @users.each do |user| %>
+    <li>
+      <%= link_to user.name, users_index_path %>
```
Comment (RGBD) [inline app/views/users/index.html.erb]: Please explain where this link leads to

Comment (sametsJR) [reply app/views/users/index.html.erb]: Along with this: get 'users / index' in routes, it displays a list of all users. You told me to do it
Screenshot from 2019-08-25 22-34-54

**Pull request merged**